src/main/java/dumshenko/daniil/Shape.java

```diff
@@ -0,0 +1,26 @@
+package dumshenko.daniil;
+
+enum FigureType {
+    CIRCLE,
+    RECTANGLE,
+    SQUARE
+}
+
+public class Shape {
+    FigureType typeOfFigure;
+    public double figureArea = 0;
```

Comment:
nit: Let's just call it `area` and you don't need to init it here I think.
```suggestion
    public double area;
```